tool/src/java/org/sakaiproject/lessonbuildertool/tool/beans/SimplePageBean.java

```diff
@@ -7473,4 +7492,40 @@ public void uploadLatestResourceVersion(){
 		}
 	}
 
+	/**
+	 * To add twitter timeline with given parameters in a Lessons page
+	 */
+	public String addTwitterTimeline(){
+		//if username is not provided return
+		if(StringUtils.isBlank(twitterUsername)){
+			return "failure";
+		}
+		//if user has added @ symbol with the username, remove it
+		if( twitterUsername.contains("@")){
+			twitterUsername = StringUtils.remove(twitterUsername, "@");
+		}
+		String href  = "https://twitter.com/" + twitterUsername;
+		//Note: widget id used is from weblearn's twitter account
```

Comment:
I meant the widget ID, isn't that something that each deployment of Sakai will want/need to configure?